src/Time/TimeSteppers/AdamsBashforthN.hpp

```diff
@@ -606,4 +641,21 @@ std::vector<double> AdamsBashforthN::get_coefficients(
   return get_coefficients_impl(steps);
 }
 
+template <typename Iterator>
+std::vector<double> AdamsBashforthN::get_coefficients(
+    const Iterator& times_begin, const Iterator& times_end,
+    const double step) noexcept {
+  ASSERT(times_begin != times_end, "No history provided");
+  std::vector<double> steps;
+  // This may be slightly more space than we need, but we can't get
```

Comment:
very helpful comment :+1: Thank you :)

Reply:
It's copy-pasted from the other version of the function, but I'm pretty sure I wrote the one there so I'll accept the complement.